tests/test_multimedia_image_api.py

```diff
@@ -20,22 +22,29 @@ class TestMultimediaAPI(IIIFTestCase):
 
     """Multimedia Image API test case."""
 
-    def setUp(self):
+    @pytest.mark.parametrize("width, height",
+                             [(1280, 1024),
+                              (100, 100),
+                              (1024, 1280),  # portrait
+                              (200, 100),
+                              (1280, 720),
+                              ])
+    def setUp(self, width=1280, height=1024):
```

Comment:
since you parametrize it, could you please move  the imports outside :)

```python
from PIL import Image
from flask_iiif.api import MultimediaImage
```